curriculum/challenges/english/16-the-odin-project/top-basic-function-projects/top-basic-functions-exercise-a.md

```diff
@@ -0,0 +1,61 @@
+---
+id: 6619240f46cec8e04d77e03a
+title: Basic Functions Exercise A 
+challengeType: 1
+dashedName: top-basic-functions-exercise-a
+---
+
+# --description--
+
+Create a function that takes in an integer. This function should return the given interger + 7 if the integer is less than 10, if the integer is greater than or equal to 10, it should return the given integer - 3.
+
+The name of the function should be `addOrSubtract`.
+
+# --hints--
+
+You should have a function called `addOrSubtract`.
+
+```js
+assert.strictEqual(addOrSubtract(5), 12);
+```
+
+Your function should take in an integer as an argument.
+
+```js
+assert.strictEqual(addOrSubtract(10), 7);
```

Comment:
this should test that camper provided a parameter for the function.
otherwise, they will be confused if they write this and are still failing the test

```js
function addOrSubtract(num){

}
```